test/org/javarosa/test/utils/ResourcePathHelper.java

```diff
@@ -0,0 +1,11 @@
+package org.javarosa.test.utils;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class ResourcePathHelper {
+    /** Makes a Path for a resource file */
+    public static Path r(String filename) {
```

Comment:
I really like moving the forms within the package under resources, but why not go “all the way to town”?

```
/** Makes a Path for a resource file */
public static Path r(String filename) {
    return Paths.get("resources", filename);
}

/**
 * Makes a Path for a resource file in the package identified by <code>classForPackage</code>
 * @param classForPackage the class whose path is to be used to locate the file in the resources directory
 * @param filename the name of the file in the subdirectory of the resources directory
 * @return a Path
 */
public static Path r(Class<?> classForPackage, String filename) {
    return Paths.get("resources", classForPackage.getPackage().getName().replace('.', '/'), filename);
}
```

Call like this:
`r(getClass(), "repeat-group-with-position-calculation.xml")`

Reply:
I also like `r(this, ...)` because it’s shorter, but some might be bothered (myself included) that far more than needed was passed as an argument.

Reply:
@dcbriccetti Actually I thought about this but wanted to use `Thread.currentThread().getStackTrace()` to automatically infer the package of the calling class. I'm fine with both approaches.

Reply:
>getStackTrace

Oh, that’s the best! It hadn’t occurred to me.

Reply:
@dcbriccetti Done 😃